.github/actions/derive-artefact-address/action.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Collect artefact address action"
+description: "Collects the artefact sub directory and commit has for pipeline being executed"
+
+inputs:
+  tag:
+      description: "The git tag to derive artefact location from"
+      required: true
+outputs:
+  commit_hash:
+    description: "Commit hash for head of branch being processed"
+    value: ${{ steps.artefact-address.outputs.commit_hash }}
+  artefact_sub_dir:
+    description: "The unique commit hash for artefacts for flows triggered by a pull request "
```

Comment:
Is this right ? Is this better described as the parent directory for the built artefacts... or something similar

Reply:
have updated description to suit the use better